proposals/lip-0063.md

```diff
@@ -374,10 +396,7 @@ def addTokenModuleEntry():
     tokenObj.userSubstore = createUserSubstoreArray()
     tokenObj.supplySubstore = createSupplySubstoreArray()
     tokenObj.escrowSubstore = []
-    tokenObj.availableLocalIDSubstore = {}
-    tokenObj.availableLocalIDSubstore.nextAvailableLocalID = nextLexicographicalOrder(LOCAL_ID_LSK)
-    # Where the function nextLexicographicalOrder returns the next byte value, in lexicographical order
-    # this is equivalent to doing localID + 1 if the byte value is viewed as a big endian integer.
+    tokenObj.supportedTokensSubstore = []
     data = serialization of tokenObj using genesisTokenStoreSchema
```

Comment:
We do not need to explicitly serialize the data for the genesis assets within the migrator anymore. The `genesis-block create` command seems to take care of it when generating the genesis block blob. We should adjust this accordingly.

Ref implementations:
-  https://github.com/LiskHQ/lisk-sdk/blob/74f59e1704c8c02db76d8f30671b91a0bb114bcc/examples/dpos-mainchain/config/default/genesis_assets.json
- https://github.com/LiskHQ/lisk-core/blob/1a01bae01bc876d4c5a9fe2006463d9488d2571e/config/devnet/genesis_assets.json

Reply:
So the implementation of the migrator does need to strictly follow the implementation details of the pseudo code of  this LIP. The pseudo code is only here to specify how the snapshot looks like. But how you get there is not important.